src/inverseRobot.js

```diff
@@ -1,13 +1,31 @@
 'use strict';
 
-/*
+/**
  * @param {object} robot
  *
- * @return {object}
-*/
+ * @return {object | null}
+ */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobotMap = new Map();
+  const valuesArray = [];
+
+  for (const key in robot) {
+    if (robot.hasOwnProperty(key)) {
+      const value = robot[key];
+
+      if (valuesArray.includes(value)) {
+        return null;
+      }
```

Comment:
you can simplify it - `Map.has()` -> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/has